packages/nodes-base/nodes/Note/Note.node.ts

```diff
@@ -0,0 +1,58 @@
+import { IExecuteFunctions } from 'n8n-core';
+import {
+	INodeExecutionData,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+
+export class Note implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'Note',
+		name: 'note',
+		icon: 'fa:sticky-note',
+		group: ['input'],
+		version: 1,
+		description: 'Leave notes and organize your flow. Supports markdown.',
+		defaults: {
+			name: 'Note',
+			color: '#FFD233',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
```

Comment:
can you remove 'main' from these? since it has no outputs/inputs

Reply:
Removed